### PR TITLE
Discussion: Add *quote command

### PR DESCRIPTION
**Pull request by Raqbit**
### Discussion

Adapted Maunz-IRC's *quote command for the Discord version.

I have not been able to test this since I don't have credentials to access the database, but this should work as it's basically the same code.

**TODO:**
- [ ] Display quotes in codeblock
- [ ] Create reactions-based system
- [ ] Test

### CODE REVIEW

**Comment by Vauff** [comment]
Make sure you keep the different import categories seperated with new lines (e.g. D4J imports will be in a different section than java imports).

I also pushed existing v2.0 progress to the v2.0-dev branch and set this PR to target it, please resolve the merge conflicts.

**Comment by Raqbit** [comment]
Fixed my styling, did some rebasing and everything should be like it's supposed to be.

**Comment by Vauff** [comment]
You need to add mysql-connector-java as a dependency, otherwise it can't find an appropriate driver when calling Util.sqlConnect()

https://gist.github.com/Vauff/610f4f0a47995c2bf066813cbc1d3aa5

File: src/com/vauff/maunzdiscord/core/Util.java [hunk 1]
```diff
@@ -124,6 +128,24 @@ public static String getOrdinal(int n)
 		}
 	}
 
+
+	public static void sqlConnect() throws Exception
+	{
+		try
+		{
+			sqlCon = DriverManager
```
Comment (Vauff) [inline src/com/vauff/maunzdiscord/core/Util.java]: Keep all of this on one line.

Comment (Vauff) [inline src/com/vauff/maunzdiscord/core/Util.java]: Keep all of this on one line.

**Comment by Vauff** [comment]
#4 has replaced ICommand with AbstractCommand, please update this PR accordingly.

**Pull request merged**